src/model/enums/piece-type.ts

```diff
@@ -5,6 +5,7 @@ export enum PieceType {
   GRAPHIC = 'GRAPHIC',
   SERVER = 'SERVER',
   AUDIO = 'AUDIO',
-  TRANSITION = 'TRANSITION'
+  TRANSITION = 'TRANSITION',
+  VIDEO_MIXER = 'VIDEO_MIXER'
```

Comment:
It seems weird that a Piece has the type of Video Mixer. The Piece type should reflect what it content it controls, not the which playout device.

Reply:
Not sure if changes are needed for this comment. 

Lindved was the one who said that a `Video_Mixer` PieceType was more correct than the originally used `Graphic` PieceType. in the `Tv2AtemVideoMixerTimelineFactory`.

Reply:
> Not sure if changes are needed for this comment.

They are. We don't have Pieces that are VideoMixers. The PieceTypes that are send to a VideoMixer are types like CAMERA and LIVE.
The PieceType reflects what kind of Piece it is. E.g. if it's a Camera or if it's a grahic or an Replay. It does not represent what hardware device is being sent to.

To sum up: There should not be a `VIDEO_MIXER` attribute on `PieceType`.

Reply:
You are currently using this attribute for creating the DskActions. These Actions are a bit special compared to our other Actions because they themselves don't do anything visual. They aren't a grahic or a camera that we can put on air. 
Instead, they just send a command to a device. In this case I think it's fine to use `PieceType.UNKNOWN`.

Reply:
Removed `VIDEO_MIXER`, and changed used PieceType for Dsk to `UNKNOWN`.